tests/Kernel/NewsRenderTest.php

```diff
@@ -0,0 +1,204 @@
+
+    $crawler = new Crawler($html);
+
+    // Body wrapper.
+    $body_wrapper = $crawler->filter('.ecl-editor');
+    $this->assertCount(1, $body_wrapper);
+    $this->assertContains('Body', $body_wrapper->text());
+
+    // Featured media.
+    $image_wrapper = $crawler->filter('article.ecl-u-type-paragraph picture img.ecl-u-width-100.ecl-u-height-auto');
+    $this->assertCount(1, $image_wrapper);
+
+    // Related links.
+    $related_links_heading = $crawler->filter('.ecl-u-type-heading-2');
+    $this->assertContains('Related links', $related_links_heading->text());
+    $related_links = $crawler->filter('.ecl-list .ecl-link.ecl-link--standalone');
+    $this->assertCount(2, $related_links);
+    $link_one = $related_links->first();
+
+    $this->assertContains('/node', trim($link_one->extract(['href'])[0]));
+    $link_two = $related_links->first();
+    $this->assertEquals('/node', trim($link_two->extract(['href'])[0]));
+  }
+
+  /**
+   * Tests News node type rendered as teaser.
+   */
+  public function testNewsTeaser(): void {
+    $file = file_save_data(file_get_contents(drupal_get_path('theme', 'oe_theme') . '/tests/fixtures/example_1.jpeg'), 'public://example_1.jpeg');
+    $file->setPermanent();
+    $file->save();
+
+    $media = Media::create([
+      'bundle' => 'image',
+      'name' => 'test image',
+      'oe_media_image' => [
+        'target_id' => $file->id(),
+        'alt' => 'Alt',
+      ],
+    ]);
+    $media->save();
+
+    $node = $this->nodeStorage->create([
+      'type' => 'oe_news',
+      'title' => 'Test news node',
+      'oe_teaser' => 'Teaser',
+      'oe_summary' => 'Summary',
+      'body' => 'Body',
+      'oe_news_featured_media' => [
+        [
+          'target_id' => (int) $media->id(),
+        ],
+      ],
+      'oe_publication_date' => '2019-04-02',
+      'oe_subject' => 'http://data.europa.eu/uxp/1000',
+      'oe_author' => 'http://publications.europa.eu/resource/authority/corporate-body/COMMU',
+      'oe_content_content_owner' => 'http://publications.europa.eu/resource/authority/corporate-body/COMMU',
+      'status' => 1,
+    ]);
+    $node->save();
+
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+    $assert = new ListItemAssert();
+
+    $expected_values = [
+      'title' => 'Test news node',
+      'url' => '/node/1',
+      'detail' => 'Teaser',
+      'meta' => 'News | 2 April 2019',
+      'image' => [
+        'src' => 'example_1.jpeg',
+        'alt' => '',
+      ],
+    ];
+    $assert->assertPattern($expected_values, $html);
+    $assert->assertVariant('thumbnail_primary', $html);
+
+    // Set news type.
+    $node->set('oe_news_types', 'http://publications.europa.eu/resource/authority/resource-type/PRESS_REL')->save();
+    $this->nodeViewBuilder->resetCache();
+
+    $build = $this->nodeViewBuilder->view($node, 'teaser');
+    $html = $this->renderRoot($build);
+
+    $expected_values = [
```

Comment:
When setting the expected values, the only one that changes is "meta", so you can just do     $expected_values['meta'] = "Press release\n | 2 April 2019";